plugin.video.slingtv/addon.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<addon id="plugin.video.slingtv" version="1.0.0" name="Sling.TV" provider-name="Lunatixz, eracknaphobia, d21spike">
+    <requires>
+        <import addon="xbmc.python" version="2.26.0"/>
+        <import addon="script.module.simplecache" version="1.0.0"/>
+        <import addon="script.module.requests" version="2.7.0"/>
+        <import addon="script.module.pytz" version="2014.2"/>
+        <import addon="script.module.beautifulsoup4" version="4.3.2"/>
+        <import addon="script.module.uepg" version="1.0.5"/>
+        <import addon="script.module.mechanize" version="0.2.6"/>
+        <import addon="script.module.inputstreamhelper" version="0.3.3"/>
+        <import addon="script.module.requests_oauthlib" version="0.7.0"/>
+        <import addon="script.module.xmltodict" version="0.11.0"/>
+    </requires>
+    <extension point="xbmc.python.pluginsource" library="default.py">
+        <provides>video</provides>
+    </extension>
+    <extension point="xbmc.addon.metadata">
+        <description lang="en_GB"></description>
+        <platform>all</platform>
+        <license>GNU GENERAL PUBLIC LICENSE. Version 3, June 2007</license>
+        <source>https://github.com/Lunatixz/plugin.video.slingtv</source>
```

Comment:
This repo doesn't exist.

Reply:
Odd, it's a valid URL; however, it's marked private.

Reply:
Guess, private repos return 404's